.github/workflows/ci.yml

```diff
@@ -104,7 +104,7 @@ jobs:
     - name: Install Rust
       uses: dtolnay/rust-toolchain@stable
       with:
-        toolchain: "1.81.0"  # STABLE
+        toolchain: "1.82.0"  # STABLE
```

Comment:
How did f1c663cba56e3d9a067b08725262703c78f79c05 become apart of this PR?

Reply:
renovate bot did it - it's not something I'm familiar with. Something enabled on this repo that automatically makes changes to PRs?

Reply:
I'm unsure how this ended up being added to the PR, but I've rebased on master and it is no longer there.